bundle/Tab/LocationView/TranslationsTab.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\BetterIbexaAdminUIBundle\Tab\LocationView;
+
+use Ibexa\AdminUi\Tab\LocationView\TranslationsTab as IbexaTranslationsTab;
+use Ibexa\AdminUi\UI\Dataset\DatasetFactory;
+use Ibexa\Contracts\Core\Repository\LanguageService;
+use Ibexa\Contracts\Core\Repository\PermissionResolver;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Data\Content\ContentUpdateAlwaysAvailableData;
+use Netgen\Bundle\BetterIbexaAdminUIBundle\Form\FormFactory;
+use Netgen\Bundle\BetterIbexaAdminUIBundle\Form\Type\Content\Translation\AlwaysAvailableUpdateType;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Form\FormFactoryInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Twig\Environment;
+
+final class TranslationsTab extends IbexaTranslationsTab
+{
+    private FormFactoryInterface $formFactory;
+    private PermissionResolver $permissionResolver;
+    private FormFactory $localFormFactory;
+
+    public function __construct(
```

Comment:
Why not use the three arguments we add here with property promotion?

Reply:
Updated in https://github.com/netgen/better-ibexa-admin-ui/pull/4/commits/7ef171c617b2045e3b292b53a0cee633e7b62546 (with unused property from the extended class removed).